public/app/core/components/ForgottenPassword/ForgottenPassword.test.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { asyncScheduler, scheduled, throwError } from 'rxjs';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { ForgottenPassword, SendResetEmailDTO } from './ForgottenPassword';
+import { backendSrv } from '../../services/backend_srv';
+import { toAsyncOfResult } from '../../../features/query/state/DashboardQueryRunner/testHelpers';
+import { createFetchResponse } from '../../../../test/helpers/createFetchResponse';
+
+jest.mock('@grafana/runtime', () => ({
+  ...((jest.requireActual('@grafana/runtime') as unknown) as object),
+  getBackendSrv: () => backendSrv,
+}));
+
+function getTestContext() {
+  jest.clearAllMocks();
+  const fetchMock = jest.spyOn(backendSrv, 'fetch');
+  const { rerender } = render(<ForgottenPassword />);
+
+  return { fetchMock, rerender };
+}
+
+describe('ForgottenPassword', () => {
+  describe('when mounted', () => {
+    it('then it should show input field', () => {
+      getTestContext();
+
+      expect(screen.getByRole('textbox')).toBeInTheDocument();
+    });
+
+    it('then it should show send button', () => {
+      getTestContext();
+
+      expect(screen.getByText(/send reset email/i)).toBeInTheDocument();
+    });
+
+    it('then it should show back to login link', () => {
+      getTestContext();
+
+      expect(screen.getByText(/back to login/i)).toBeInTheDocument();
```

Comment:
```suggestion
      expect(screen.getByRole('link', { name: /back to login/i })).toBeInTheDocument();
```
Link button can be matched via `link` role, as it's an `a` element. The `name` in this case will be the actual content inside the link.

Reply:
Fixed!